templates/allauth/account/account_inactive.html

```diff
@@ -0,0 +1,13 @@
+{% extends "account/base.html" %}
+
+{% load i18n %}
+
+{% block head_title %}{% trans "Account Inactive" %}{% endblock %}
```

Comment:
Again a use of translation - is the intention to do translation across the entire project?
If not, I think it's better to avoid it entirely in this project, and perhaps teach it with a smaller toy project.

Reply:
all these templates are builtin to allauth - these are just copies of the originals with a few tweaks, so the translation here is coming from allauth